packages/token-list/src/index.js

```diff
@@ -139,7 +137,7 @@ export default class TokenList {
 	 * @see https://dom.spec.whatwg.org/#dom-domtokenlist-toggle
 	 *
 	 * @param {string}   token Token to toggle.
-	 * @param {?boolean} force Presence to force.
+	 * @param {boolean=} force Presence to force.
```

Comment:
There are two ways to mark optional params, should we enforce somehow one of them?

@param {number} [foo]
// or:
@param {number=} foo

I think I saw the first in our codebase before. The one you proposed I had to check in docs 😃

Just asking while I learn, as long it’s valid it isn’t a blocker but I think we were opinionated in other places for consistency.

Reply:
> There are two ways to mark optional params, should we enforce somehow one of them?

Definitely need some standards here. I added to consider in [this week's chat agenda](https://docs.google.com/document/d/1TJ06NiyLzThsplXdEhya51C13WE62nHN71kt997hXHw/edit).

They both appear to be valid:

https://jsdoc.app/tags-param.html#optional-parameters-and-default-values

And both work in TypeScript from my testing.

I don't feel too strongly about it, but there are two minor advantages I see in the `=` suffix:

- Since this is a detail about the type ("or undefined"), it makes sense to include this in the type segment
   - This could also make type parsing for Doctrine replacement (#18045) easier if we need to start implementing this ourselves, as otherwise we need to reach into the parsed "name" to have a complete understanding of the type.
- When we're fighting against 80 (or 100) column line limit, we might like to have the extra one character per line for every line in the parameters grouping 😄

Reply:
I copied examples from the docs you linked :)

`@param {number=} foo` would be my pick based on the argument about parsing docs if you cal always provide the default value with such syntax:

`@param {number=5} foo`?

Reply:
But I also wonder if there's much value in that syntax these days, considering it's typically redundant with [default function parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters), which are equally capable to be statically analyzed by tooling.

Reply:
While I still think it could be something to revisit, given the ability of static analysis to understand a default from the function parameter signature, I later found that this syntax is already codified in the JavaScript Documentation Standards:

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#functions

(See optional parameters examples)

For this pull request, I'll just adhere to that standard.

Reply:
Yes, let's follow the existing docs for now. We can propose the change later 👍 

Thanks for sharing this discovery 😄